pkg/rpc/server/server_helper_test.go

```diff
@@ -32,6 +31,11 @@ func initServerWithInMemoryChain(t *testing.T) (*core.Blockchain, *Server, *http
 
 	go chain.Run()
 
+	return chain, cfg, logger
+}
+
+func getTestBlocks(t *testing.T) []*block.Block {
+	blocks := make([]*block.Block, 0)
```

Comment:
Why not allocate right after the `nBlocks` initialization?

Reply:
OK, why not.